backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultUploadService.java

```diff
@@ -110,6 +112,10 @@ public TestResultUpload processResultCSV(InputStream csvStream) {
         testResultFileValidator.validate(new ByteArrayInputStream(content));
     if (!errors.isEmpty()) {
       validationErrorResult.setErrors(errors.toArray(FeedbackMessage[]::new));
+
+      errorRepository.saveAll(
+          errors.stream().map(error -> new ResultUploadError(org, error)).toList());
```

Comment:
hmm I wonder if not having a common id for all these errors will make it harder for us to be able to say they were all part of the same file upload, I guess right now we would have to go by the date and org id unique combo, wonder of its worth generating an extra field (submission_id) with all of them having the same random uuid.

Or we can change the ResultUploadError.upload_id to a UUID on the object, rather than the TestResultUpload, that way we can just fill it with the same random uuid for that particular submission when failing SR migration. aka. Remove the key constraint. if we do that, I would also advocate for doing the same to the org_id

Reply:
> Or we can change the ResultUploadError.upload_id to a UUID on the object, rather than the TestResultUpload

Would we lose the ability to correlate an individual error to its `upload`? Do we even care?

> I would also advocate for doing the same to the org_id

What do you mean by this?

Reply:
Yea honestly, I wanna push for a `submission_id` to correlate two submissions as part of the same upload. and I do like the source field that was added

to clarify my comments about org_id, and upload_id,I meant just changing them to UUID

```
public class ResultUploadError extends AuditedEntity {
  @Column(name = "upload_id")
  private UUID upload;

  @Column(name = "org_id")
  private UUID organization;
```

just as optimization, but we dont have too